server/resolvers/TestPlanReportOperations/assignTesterResolver.js

```diff
@@ -0,0 +1,22 @@
+const { AuthenticationError } = require('apollo-server');
+const {
+    createTestPlanRun
+} = require('../../models/services/TestPlanRunService');
+
+const assignTesterResolver = async (
+    { parentContext: { id: testPlanReportId } },
+    { user: testerUserId },
+    { user }
+) => {
+    if (!user.roles.includes('ADMIN')) {
```

Comment:
nit-pick: Could probably have some global-level constants defined for the roles just as some typing safety, chance of any to the role name (unlikely), etc - if they'll be referenced often. Could also be defined directly on the `User` model

Reply:
So something like `user.roles.includes(User.ADMIN)`? Yes, I'd be willing to try that!

Reply:
Yep, exactly that. Cool!

Reply:
This will be in my next PR.